k8s/serviceinstance/loop.go

```diff
@@ -111,24 +114,43 @@ func handleAddServiceInstance(
 		InstanceID:        serviceInstance.Spec.ID,
 		ServiceID:         sc.ID,
 		PlanID:            serviceInstance.Spec.PlanID,
-		AcceptsIncomplete: false,
+		AcceptsIncomplete: true,
 		Parameters:        serviceInstance.Spec.Parameters,
 	}
 	finalServiceInstanceState := data.ServiceInstanceStateFailed
+	finalServiceInstanceStateReason := ""
 	defer func() {
```

Comment:
you'll have to update the service instance's resource version before you call the `updateFn` below, otherwise it'll fail